chart/kubeapps/Chart.yaml

```diff
@@ -31,4 +31,4 @@ maintainers:
 name: kubeapps
 sources:
   - https://github.com/vmware-tanzu/kubeapps
-version: 11.0.2-dev1
+version: 11.1.0-dev0
```

Comment:
Let's see who gets there first (you and Rafa are both updating in your PRs :) ).